test/Directory.Build.targets

```diff
@@ -1,4 +1,5 @@
 <Project>
+  <Import Project="../Directory.Build.targets" />
```

Comment:
I think we could use`$(RepoRoot)\Directory.Build.targets` to keep "standard" ref to repo root.

Reply:
I believe climbing up the directory structure manually is preferable over using an absolute path. Just my 5c.

Reply:
Thank's @ViktorHofer for advice, it's only a "stylistic" choice or could lead in future to some issue?

Reply:
Purely stylistic

Reply:
Changes it to ```$(RepoRoo)``` style.